app/src/main/java/com/example/parkingsystemkotlin/mvp/contract/ReservationParkingContract.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.parkingsystemkotlin.mvp.contract
+
+interface ReservationParkingContract {
+
+    interface ReservationActivityPresenter {
+        fun onButtonStartPressedSelectReserver()
+        fun onButtonEndPressedSelectReserver()
```

Comment:
minor change: include in the name of this method something related to "Date". For example: onButtonStartDatePressedSelection

Reply:
Done